pkg/rotate/rotatekeys.go

```diff
@@ -255,9 +257,9 @@ func updateKeyLocation(account string, keyLocations config.KeyLocations, keyID,
 
 	// all done
 	logger.Infow("Key locations updated",
```

Comment:
I *think* you can attach a String function to KeyWrapper that would allow you to just print the struct from here, e.g.

func (kw KeyWriter) String() string {
        return fmt.Sprintf("KeyWrapper: key=%s", kw.Key) // etc.
}

Reply:
Actually my bad - you're logging more than just the KeyWrapper, so I think it's fine as it is.